### PR TITLE
Add exit button to collapse metadata preview

### PR DESCRIPTION
**Pull request by nunezco2**
A close button on the top right side of the metadata window has been added to close the preview for the currently selected entity.

### CODE REVIEW

**Review by longshuicy — APPROVED** [review]
Looks good to me, approved!

**Review by longshuicy — APPROVED** [review]

File: src/components/DataViewer.jsx [hunk 1]
```diff
@@ -109,6 +109,9 @@ const styles = {
 	previewClose: {
 		display: "inline",
 		float: "right"
+	},
+	metadataCloseButton:{
+		float:"right",
```
Comment (navarroc) [inline src/components/DataViewer.jsx:114]: This is minor, but can you add a space after the colon?

Comment (navarroc) [reply src/components/DataViewer.jsx:114]: Other viewers have the same minor issue.

**Review by ywkim312 — APPROVED** [review]
worked for me. Approved

**Pull request merged**